plugins/github-commit-status.js

```diff
@@ -1,10 +1,10 @@
 var util   = require('util'),
-Git        = require('../lib/git'),
-GitHubApi  = require('github');
```

Comment:
If we're dropping that `github` module, can we remove it? Or is it used elsewhere... checking now...

Reply:
It's not used elsewhere, I think I dropped it from the `package.json`

Reply:
!

Oh, shoot. NVM. I thought this was importing the _plugin_. All good. :+1: